sdk/datafactory/arm-datafactory/src/dataFactoryManagementClient.ts

```diff
@@ -55,7 +54,6 @@ class DataFactoryManagementClient extends DataFactoryManagementClientContext {
     this.activityRuns = new operations.ActivityRuns(this);
     this.triggers = new operations.Triggers(this);
     this.triggerRuns = new operations.TriggerRuns(this);
-    this.rerunTriggers = new operations.RerunTriggers(this);
```

Comment:
>RerunTriggers [](start = 40, length = 13)

this is a breaking change, right? is this because a new api version is used?

Reply:
Yes. This is a breaking change. I have updated the package version.  I think the rerunTriggers has been deleted in the swagger